requirements/base.txt

```diff
@@ -394,7 +394,7 @@ werkzeug==3.0.3
     #   flask-login
 wrapt==1.16.0
     # via deprecated
-wtforms==3.1.2
+wtforms==3.2.1
```

Comment:
Should these bumps be done in a separate PR? They don't have a history of going super smoothly.